.gitignore

```diff
@@ -1,3 +1,6 @@
 node_modules
 npm-debug.log
 .DS_Store
+.idea/
+.npmrc
```

Comment:
.npmrc is used on `folio-web` to save Nexus repository URL and credentials placeholder.

Reply:
Yes, that's how I'm using it locally, but we don't want credentials stored in git, hence why it should be added to .gitignore.

Reply:
This is what we have on our git repositories of other JS code:
```
registry=https://repo.helix.re/repository/helix-npm/
always-auth=true
_auth=${NEXUS_AUTH}
email=${NEXUS_EMAIL}
```
I'm not sure if it is used by our CI pipelines to build the code, but that is what we have.
If there is no use of this info for the project, I believe it is ok to add it to `.gitignore`.

Reply:
Yes @heryxpc, `.npmrc` is safe to push to Git as no credentials are stored in the file (unless you don't want the registry path to be visible publicly).